tests/error/linear_errors/break_unused.py

```diff
@@ -26,7 +26,7 @@ def foo(i: int) -> bool:
         if i == 0:
             break
         i -= 1
-        b ^= measure(q)
+        b &= measure(q)
```

Comment:
for some reason `bool.__xor__` is missing (see #274).

The test worked before since the linearity violation triggered an error before we got to this point. Now we type check everything first, so this has to be fixed (similar for the remaining modified tests)